lib/krane/deploy_task.rb

```diff
@@ -8,5 +8,9 @@ def initialize(**args)
       raise "Use Krane::DeployGlobalTask to deploy global resources" if args[:allow_globals]
       super(args.merge(allow_globals: false))
     end
+
+    def prune_whitelist
```

Comment:
are we adding this here (instead of updating `Krane::DeprecatedDeployTask`) to avoid changing `KubernetesDeploy::DeployTask`?

Reply:
Yes